scripts/install/panel.sh

```diff
@@ -10,48 +10,59 @@
 #   under the GPL along with build & install instructions.
 #
 #! /bin/bash
-if [[ -f /tmp/.install.lock ]]; then
-  log="/root/logs/install.log"
-else
-  log="/root/logs/swizzin.log"
-fi
 
 if [[ ! -f /install/.nginx.lock ]]; then
-  echo "This package requires nginx to be installed!"
-  read -p "Press enter to proceed with installing nginx before the panel."
-  bash /usr/local/bin/swizzin/install/nginx.sh
+  echo_warn "This package requires nginx to be installed!"
+  if ask "Install nginx?" Y; then
+    bash /usr/local/bin/swizzin/install/nginx.sh
+  else
+    exit 0
```

Comment:
I believe this should be non-zero since by exiting we did not complete the action we set out to do.